services/port_manager/src/main/java/com/futurewei/alcor/portmanager/proxy/RouteManagerProxy.java

```diff
@@ -45,23 +46,25 @@ public RouteManagerProxy(Stack<Rollback> rollbacks) {
     public PortBindingRoute getRouteBySubnetId(Object arg1, Object arg2) throws Exception {
         String portId = (String)arg1;
         String subnetId = (String)arg2;
-        RouteWebJson routeWebJson = routeManagerRestClient.getRouteBySubnetId(subnetId);
-        if (routeWebJson == null || routeWebJson.getRoute() == null) {
+        RoutesWebJson routesWebJson = routeManagerRestClient.getRouteBySubnetId(subnetId);
+        if (routesWebJson == null || routesWebJson.getRoutes() == null) {
             throw new GetRouteEntityException();
         }
 
-        return new PortBindingRoute(portId, routeWebJson.getRoute());
+        // FIXME : we should bind a port with several route rules
+        return new PortBindingRoute(portId, routesWebJson.getRoutes().get(0));
```

Comment:
Create Issue #255 for tracking.

Reply:
Thanks Liguang!

Reply:
Np. Let us keep this comment unresolved to remind us.